bundles/org.openhab.binding.deconz/README.md

```diff
@@ -147,7 +147,7 @@ Other devices support
 | brightness        | Dimmer                   |     R/W     | Brightness of the light               | `dimmablelight`                               |                                 
 | switch            | Switch                   |     R/W     | State of a ON/OFF device              | `onofflight`                                  |
 | color             | Color                    |     R/W     | Color of an multi-color light         | `colorlight`, `extendedcolorlight`            |
-| color_temperature | Number                   |     R/W     | `0`->`100` represents cold -> warm    | `colortemperaturelight`, `extendedcolorlight` |
+| color_temperature | Number                   |     R/W     | Color temperature in kelvin           | `colortemperaturelight`, `extendedcolorlight` |
```

Comment:
Maybe add a note below that the Color temperature range depends on the individual light.

Reply:
good point will do